bin/pycbc_inspiral

```diff
@@ -223,6 +223,7 @@ with ctx:
         'bank_chisq'     : float32,
         'bank_chisq_dof' : int,
         'cont_chisq'     : float32,
+        'qtransform'     : dict,
```

Comment:
As of right now, no. But it also doesn't affect the output of the code in any detrimental way.